presto-docs/src/main/sphinx/develop/presto-native.rst

```diff
@@ -159,3 +159,25 @@ Async Data Cache and Prefetching
 
     Size of the SSD cache when async data cache is enabled.  Must be zero if
     ``async-data-cache-enabled`` is ``false``.
+
+``enable-old-task-cleanup``
+^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type** ``bool``
+    * **Default value:** ``true``
```

Comment:
```suggestion
    * **Default value:** ``true``
    * **Presto on Spark default value:** ``false``
```
Suggested formatting for consistency with ``num-connector-io-threads`` and ``async-data-cache-enabled`` entries, also on this same page. This page isn't published in the current doc yet so I'm including a screenshot. 
<img width="1018" alt="Screenshot 2023-10-20 at 10 29 07 AM" src="https://github.com/prestodb/presto/assets/7013443/5474be32-31b6-4ee0-b631-78eb540ddeb8">